core/pom.xml

```diff
@@ -29,6 +29,88 @@
                     <skipDeploy>true</skipDeploy>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
Lots of discussion earlier with Ale, we found that --release is the issue. That's not all 

I found that basically we need to override every exevery execution id that is defined in parent pom and redefine it here in mule-core.

@elrodro83 Now default-compile compiles everything with **java 8** and base-compile compiles everything with **Java 11** and module-compille compilea module-info. This works and builds mule-core, jar seems ok and has JarUtils class in inside versions.11.metainf but seems versions 11 , need to check if it works. Atleast compile is ok. We may need to rename the maven-compille-plugin ids in mule parent pom.